src/custom/pages/Claim/InvestmentFlow/index.tsx

```diff
@@ -244,18 +251,8 @@ export default function InvestmentFlow({ hasClaims, isAirdropOnly, modalCbs }: I
           </ClaimTable>
 
           <AccountClaimSummary>
-            <AccountDetails
-              label="Claiming with account"
-              account={account}
-              connectedAccount={account}
-              Icon={DollarSign}
-            />
-            <AccountDetails
-              label="Receiving account"
-              account={activeClaimAccount}
-              connectedAccount={account}
-              Icon={Send}
-            />
+            <AccountDetails isClaimer label="Claiming with account" account={account} connectedAccount={account} />
+            <AccountDetails label="Account receiving vCOW" account={activeClaimAccount} connectedAccount={account} />
```

Comment:
And here also a bit shorter:

```suggestion
            <AccountDetails label="Receiving on" account={activeClaimAccount} connectedAccount={account} />
```

The we'll have:

Claiming with 0x000...
Receiving on 0x000...

Reply:
Yes, good with me 👍🏼